packages/platform/platform-http/src/common/services/PlatformAdapter.ts

```
@@ -53,7 +52,9 @@
   /**
    * Map handler to the targeted framework
    */
-  abstract mapHandler(handler: Function, layer: PlatformHandlerMetadata): Function;
+  mapHandler(handler: Function, layer: PlatformHandlerMetadata): Function {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)